aws_lambda_powertools/utilities/idempotency/persistence/base.py

```diff
@@ -19,6 +19,7 @@
     IdempotencyItemAlreadyExistsError,
     IdempotencyValidationError,
 )
+from aws_lambda_powertools.utilities.validation.jmespath_functions import PowertoolsFunctions
```

Comment:
Could you leave a comment as `Maintenance: Make PowertoolsFunctions a shared class`? That way we can improve it later

Reply:
@heitorlessa i moved it into the shared dir.